packages/fast-component-explorer/app/explorer.style.ts

```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
     },
     explorer: {},
     explorer_colorPicker: {
-        background: neutralFillRest(() => "#3B3B3B"),
+        background: neutralFillRest(neutralLayerL2),
         border: "none",
         width: "23px",
+        height: "22px",
         ...applyCornerRadius(),
         "&:hover": {
-            background: neutralFillHover(() => "#3B3B3B"),
+            background: neutralFillHover(neutralLayerL2),
         },
         "&:active": {
-            background: neutralFillActive(() => "#3B3B3B"),
+            background: neutralFillActive(neutralLayerL2),
         },
         "&:focus": {
             outline: "none",
         },
     },
-    explorer_viewerRegion: {
+    explorer_devToolsPanel: {
+        boxShadow:
+            "0px 0.6px 1.8px rgba(0, 0, 0, 0.11) inset, 0px 3.2px 7.2px rgba(0, 0, 0, 0.13) inset",
```

Comment:
Are these custom, or are these related to elevation?

Reply:
Custom, the shadows need to come from specific directions.